libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotification.kt

```diff
@@ -93,15 +97,15 @@ class MapboxTripNotification constructor(
     private val notificationReceiver = NotificationActionReceiver()
     private val distanceFormatter: DistanceFormatter =
         navigationOptions.distanceFormatter
-            ?: throw IllegalArgumentException("Distance formatter is required.")
+            ?: throw IllegalArgumentException(MAPBOX_NAVIGATION_NOTIFICATION_FORMATTER_EXCEPTION)
```

Comment:
should we make `distanceFormatter` in `NavigationOptions ` nonNullable?

Reply:
@RingerJK the `MapboxTripNotification` is an optional dependency, so it might be hard to justify making it required by defualt.

Reply:
make sense, thanks